hal/common/file.hpp

```diff
@@ -142,14 +142,21 @@ class File
 	 *
 	 * \return 	success	(always false)
 	 */
-	 bool flush();
+	bool flush();
 
 	/**
 	 * \brief 	write newline character to file ('\n')
 	 *
 	 * \return 	success
 	 */
-	 bool newline();
+	bool newline();
+
+	/**
+	 * \brief	sync the file
```

Comment:
If you declare `flush` as virtual in `File`, then you can use it instead of adding the function `sync`